contracts/addresses/addresses.go

```diff
@@ -1,17 +1,50 @@
 package addresses
 
-import "github.com/ethereum/go-ethereum/common"
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+type CeloAddresses struct {
+	CeloTokenAddress            common.Address
```

Comment:
nit: it's a matter of taste, but since the struct is already called "..Addresses" and the type specifies that those are addresses, I would rename the field names to `CeloToken` etc.